extensions/amp-story/validator-amp-story.protoascii

```diff
@@ -864,6 +864,14 @@ tags: {  # <amp-story-page-attachment> with href
       protocol: "https"
     }
   }
+  attrs: {
+    name: "cta-image"
```

Comment:
Given the other attributes `data-cta-text` and `data-title`, I think this should be `data-cta-image`

Reply:
Yeah we had this discussion and the finding was that the api has inconsistent naming (`theme` doesn't have a `data-` prefix, for example) and given that, it will be cleaner to try to stick with not using the `data-` prefix moving forward. I know it's a bit inconsistent in the element, but now for the inline-cta elements we're using `cta-image` and `cta-image-2` without the `data-` prefixes and I think it's most important to stay consistent with that choice too (see lines 887 & 895 of this file)

Reply:
Should we file a ticket to allow `cta-text` and `title` and update the documentation, while still supporting `data-cta-text` and `data-title` to be backward compatible?

Reply:
Yes I support this!!